docs/key_rotation.md

```diff
@@ -0,0 +1,114 @@
+# Key Rotation
+> CAUTION: Please read carefully and ensure you understand these instructions. Rotating the wrong key could lock you out of your account and make funds permanently inaccessible.
+
+There are two cases:
+
+1) you are in full control of an account, and would like to
+rotate to a new private key (using a new mnemonic).
+
+This is a single step, and you can simply use the current mnemonic (to sign a
+transaction), and the new mnemonic (to sign a rotation proof).
+
+2) you are claiming an account from someone else.
+
+This requires two steps where the current owner (Alice) will first authorize an existing
+account of the new owner (Bob) to rotate keys for the account being
+claimed. The Bob will have two accounts at the end of the process, and the
+prior owner, Alice, will have none.
+
+
+## CASE 1: Rotate Keys on Your Wallet
+
+You will be prompted for a mnemonic twice. But theses should be DIFFERENT
+mnemonics.
+
+The first mnemonic is for your current credentials which will be decprected. It
```

Comment:
decprected -> deprecated

Reply:
Thanks @minaxolone. Fixed in the latest commit